drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -451,8 +451,12 @@ static inline struct usbd_event *usbd_evt_alloc(void)
 
 	if (ret < 0) {
 		LOG_ERR("USBD event allocation failed!");
+		if (k_is_in_isr()) {
```

Comment:
Could You please provide an explanation for this?

Reply:
It can not be flushed in ISR (in the issue it is usb_dc_nrfx_power_event_callback()). "The kernel does allow an ISR to remove an item from a fifo" from https://docs.zephyrproject.org/latest/reference/kernel/data_passing/fifos.html

Reply:
I misunderstood something. But flushing did not work here.

Reply:
Removed it.